src/Auction.hs

```diff
@@ -151,4 +153,9 @@ extractLastCall =
     -- from the action.
     lastCall . fst . finish T.North
 
+-- displayLastCall is for use in explanations: it formats the most recent call
+-- from an action while stripping out any alerts it might have
+displayLastCall :: OutputType -> Action -> String
+displayLastCall fmt = output fmt . T.extractCall . extractLastCall
```

Comment:
The names `extractCall` and `extractLastCall` are misleadingly similar. How about `removeAlert` for the new one? Maybe something else?